lightning/src/ln/channelmanager.rs

```diff
@@ -6409,7 +6431,7 @@ where
 					if let msgs::ErrorAction::IgnoreError = err.err.action {
```

Comment:
This method has some occurrences of `self.logger` that look like they could benefit from a contextual logger w/ a payment hash?

Reply:
mm i find it a bit tricky here because iam not able to find a way to get the channel context so I can initialize `WithChannelContext`